### PR TITLE
Fix MAX_SPDM_MEASUREMENT_BLOCK_COUNT should be 0xFE

### LINKED ISSUE #309: MAX_SPDM_MEASUREMENT_BLOCK_COUNT should be 0xFE

### PR DESCRIPTION
**Pull request by xiaoyuxlu**
- Fix #309

### CODE REVIEW

File: spdmlib/src/message/measurement.rs [hunk 1]
```diff
@@ -277,8 +278,9 @@ mod tests {
                             value: [100u8; MAX_SPDM_MEASUREMENT_VALUE_LEN],
```
Comment (jyao1) [inline spdmlib/src/message/measurement.rs:278]: I think we should remove `MAX_SPDM_MEASUREMENT_VALUE_LEN` as well.

File: spdmlib/tests/test_library.rs [hunk 2]
```diff
@@ -4,8 +4,7 @@ use codec::{Reader, Writer};
 use spdmlib::common::opaque::*;
 use spdmlib::common::SpdmCodec;
 use spdmlib::config::{
-    MAX_SPDM_CERT_CHAIN_DATA_SIZE, MAX_SPDM_MEASUREMENT_BLOCK_COUNT,
-    MAX_SPDM_MEASUREMENT_VALUE_LEN, MAX_SPDM_OPAQUE_SIZE,
+    MAX_SPDM_CERT_CHAIN_DATA_SIZE, MAX_SPDM_MEASUREMENT_VALUE_LEN, MAX_SPDM_OPAQUE_SIZE,
```
Comment (jyao1) [inline spdmlib/tests/test_library.rs:7]: We need remove `MAX_SPDM_CERT_CHAIN_DATA_SIZE`.

File: spdmlib/build.rs [hunk 3]
```diff
@@ -108,7 +107,6 @@ pub const MAX_SPDM_MEASUREMENT_VALUE_LEN: usize = {meas_val_len};
 /// This is used in SpdmMeasurementsResponsePayload
 /// There could be multiple measurements in one measurement record.
 //pub const MAX_SPDM_MEASUREMENT_RECORD_LEN : usize = 0x400;
-pub const MAX_SPDM_MEASUREMENT_BLOCK_COUNT: usize = {meas_block_cnt};
```
Comment (jyao1) [inline spdmlib/build.rs:111]: I think we need remove `MAX_SPDM_MEASUREMENT_RECORD_LEN` as well.

File: spdmlib/tests/test_library.rs (hunk 2 above)
Comment (xiaoyuxlu) [reply spdmlib/tests/test_library.rs]: removed!

File: fuzz-target/requester/challenge_req/src/main.rs [hunk 4]
```diff
@@ -27,7 +27,7 @@ fn fuzz_send_receive_spdm_challenge(fuzzdata: &[u8]) {
     responder.common.reset_runtime_info();
     responder.common.provision_info.my_cert_chain = Some(SpdmCertChainData {
         data_size: 512u16,
-        data: [0u8; config::MAX_SPDM_CERT_CHAIN_DATA_SIZE],
+        data: vec![0u8; 512],
```
Comment (jyao1) [inline fuzz-target/requester/challenge_req/src/main.rs:30]: please remove vec!

**Review by jyao1 — CHANGES_REQUESTED** [review]
Why use so many Vector?

**Comment by jyao1** [comment]
OK, I think this PR includes too many things.

Please submit different PR to fix different issue #309, #328, #329.

**Pull request closed without merging**